### ISSUE TITLE
ACF Pro Compatible?

### ISSUE THREAD

**Issue by brrn** [alexvandervegt/advanced-custom-fields-widget#3]
I love this concept, but I can't get my custom widget to appear in the Location Rules drop-down menu while using ACF Pro. Also, while using the "ACF Widget" widget, the "Select ACF Group" drop-down doesn't display any of my field groups as options. Is this a compatibility issue?

**Comment by rfverde1** [comment]
I had the same issue when trying out this module as well. I spent quite some time to figure out why they were not showing. It boiled down to compatibility. This module uses a filter to show the ACF groups:
// Get ACF field groups
$acfs = apply_filters("acf/get_field_groups", array());

ACF PRO moved away from filters and replaced it to function calls instead.

**Comment by brrn** [comment]
Ah, that makes sense.

I just learned that this plugin was adopted into ACF after reading this closed issue: https://github.com/alexvandervegt/advanced-custom-fields-widget/issues/2

I was able to solve my issue by disabling this plugin and using the widget Location Rule which is now built in to ACF. By following the documentation for this plugin, I was able to create a widget and assign fields to it without using the "ACF Widget" provided by this plugin.

Thank you!
